docusaurus/website/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "scripts": {
+    "start": "docusaurus start",
+    "build": "docusaurus build",
+    "swizzle": "docusaurus swizzle",
```

Comment:
lol, what's swizzle?

Reply:
It's some sort of setup where you can eject some hidden boilerplate out and customize it. Then if you don't like it you can just delete it and it goes back to default. Might eventually give it a try